### ISSUE TITLE
Crashloop during pregen

### ISSUE THREAD

**Issue by NLBlacky** [PG85/OpenTerrainGenerator#348]
#### OTG Plugin or Mod
- [x] OTG Mod for Forge

#### OTG Version
- [x] 1.12.2_v8_r1

#### Forge or Spigot Versions
forge-1.12.2-14.23.5.2838-universal.jar

#### Server/Client Log
[Crashlog](https://pastebin.com/Hq93kdiK)

#### What other plugins are you running?
Not sharing my plugin list, but I can confirm that without plugins it is still happening.

#### Description
While pregenning a 10k radius area, the server got stuck in a crashloop every now and then. Crashlog listed above is the crash error I got in all crashlogs. The server I was pregenning is completely fresh and didn't have any world generated by a previous OTG. I would prefer to understand the issue more and have a fix for the crashes.

**Comment by authvin** [comment]
OTG doesn't run on Sponge servers, there's a lot of bugs. That's the most likely culprit.

**Comment by NLBlacky** [comment]
I’ve run OTG with our Sponge servers for nearly 1,5 year and has worked fine for me.

**Comment by PG85** [comment]
Hey @NLBlacky , thanks for reporting this. This isn't the Sponge error we usually get from people who can't get it running at all. If you do get OTG running on a Sponge server, that would be the first report I've seen. I've fixed the nullpointerexception for 8.1_r1, to be released on the OTG discord's dev channel asap. Will be included in the next release.

**Comment by PG85** [comment]
When this is released, please let me know if it works for you btw. I bet we'll have some users curious to know how exactly you got it working with Sponge ;)

**Comment by NLBlacky** [comment]
We never had any issues with OTG on our Sponge servers for over a year now. We're currently on `spongeforge-1.12.2-2838-7.1.7-RC3924.jar`. We usually rename our Sponge jars to `aaa_spongeforge-1.12.2-2838-7.1.7-RC3924.jar` to make sure that Sponge loads first. Maybe this will solve it for many players. Beside that we don't have any changes regarding Sponge so I'm not sure why it doesn't work for others. In the past we have had some crashes over time due to a null return somewhere in the code, which we fixed by editing a line in the jar. I believe CreepyCre confronted you about that long ago, but it was fixed in your last version if I recall correctly.
